src/react-aria-modal.js

```diff
@@ -166,9 +167,9 @@ let Modal = class Modal extends React.Component {
       dialogProps.tabIndex = '-1';
     }
 
-    const childrenArray = [React.DOM.div(dialogProps, props.children)];
+    const childrenArray = [DOM.div(dialogProps, props.children)];
```

Comment:
Is there a reason to use `react-dom-factories` instead of `React.createElement('div', ..)`? Was thinking we could avoid the additional dependency.